manifests/components/cert-manager.jsonnet

```diff
@@ -44,11 +44,104 @@ local CERT_MANAGER_IMAGE = (import "images.json")["cert-manager"];
   },
 
   certCRD: kube.CustomResourceDefinition("certmanager.k8s.io", "v1alpha1", "Certificate") {
-    spec+: { names+: { shortNames+: ["cert", "certs"] } },
+    spec+: {
+      additionalPrinterColumns+: [
+        {
+          "JSONPath": ".status.conditions[?(@.type==\"Ready\")].status",
+          "name": "Ready",
+          "type": "string",
+        },
+        {
+          "JSONPath": ".spec.secretName",
+          "name": "Secret",
+          "type": "string",
+        },
+        {
+          "JSONPath": ".spec.issuerRef.name",
+          "name": "Issuer",
+          "priority": 1,
+          "type": "string",
+        },
+        {
+          "JSONPath": ".status.conditions[?(@.type==\"Ready\")].message",
+          "name": "Status",
+          "priority": 1,
+          "type": "string",
+        },
+        {
+          "description": "CreationTimestamp is a timestamp representing the server time when this object was created. It is not guaranteed to be set in happens-before order across separate operations. Clients may not set this value. It is represented in RFC3339 form and is in UTC.",
+          "JSONPath": ".metadata.creationTimestamp",
+          "name": "Age",
+          "type": "date",
+        },
+      ],
+      names+: {
+        shortNames+: ["cert", "certs"],
+      },
+    },
+  },
+
+  issuerCRD: kube.CustomResourceDefinition("certmanager.k8s.io", "v1alpha1", "Issuer") {
+    spec+: {
+      additionalPrinterColumns+: [
```

Comment:
By reading https://github.com/jetstack/cert-manager/blob/release-0.6/deploy/manifests/cert-manager.yaml (which I believe this jsonnet fragment is based on), I don't see any references to this `additionalPrinterColumns` attribute.

Reply:
good catch! I seemed to have mixed it up. Now fixed. Please take another look